requirements.txt

```diff
@@ -1,4 +1,4 @@
-cython>=0.21
-numpy>=1.12
+cython>=0.29.20
+numpy>=1.16.6,<1.20
```

Comment:
There's no upper limit on Numpy at runtime, unless something's changed recently.

Reply:
Indeed the runtime does not have upper limit, only the building requirements do. Ive just updated `requirements.txt`.